pkg/client/dtclient/settings_client.go

```diff
@@ -60,6 +117,13 @@ type SettingsModificationInfo struct {
 	NonModifiablePaths []interface{} `json:"nonModifiablePaths"`
 }
 
+type SettingsResourceContext struct {
+	Operations         []string      `json:"operations"`
+	Movable            *bool         `json:"modifications:movable"`
```

Comment:
To the reviewer. I chose a pointer over its own `struct` type, the problem here is that the param `Movable` is optional, and logic code in method `isMovable()` depends on it. 

If preferred, I could change it to a `NullBool` type and implement the unmarshaling myself. Since is not done anywhere else in the code i chose `*bool` over `NullBool`.

Reply:
By now, I wasn't aware of the NullBool type in GO. But as I see it, it is part of the SQL library and for that reason wouldn't it even consider. For me pointer is just enough